services/distribution/src/main/resources/application.yaml

```
@@ -131,6 +131,8 @@ services:
         value: ${DCC_PERSON_COUNT_MAX:20}
       - label: dcc-person-warn-threshold
         value: ${DCC_PERSON_WARN_THRESHOLD:4}
+      - label: dcc-admission-check-scenarios-disabled
+        value: ${DCC_ADMISSION_CHECK_SCENARIOS_ENABLED:0}
```

@mlenkeit this will be the same for iOS and Android - is that OK? And `0` is the right default as of now, correct?

---

@AliceBilc I would propose the following (two) changes: align the env var name with the `label` attribute and then set it to `1` by default.
```suggestion
      - label: dcc-admission-check-scenarios-disabled
        value: ${DCC_ADMISSION_CHECK_SCENARIOS_DISABLED:1}
```